be able to configure the http method used (#146)
prometheus-community/codemirror-promql: README.md

const promQL = new PromQLExtension().setComplete({ remote: { httpErrorHandler: (error: any) => console.error(error) } })
```

###### HTTP method used

> By default, the prometheus client will use the HTTP method `POST` when contacting Prometheus for the

Reviewer: ```suggestion
By default, the Prometheus client will use the HTTP method `POST` when contacting Prometheus for the
```

endpoints `/api/v1/labels` and `/api/v1/series`.

You can change it to use the HTTP method `GET` if you prefer.

```typescript

> const promQL = new PromQLExtension().setComplete({ remote: { httpMethodUsed: 'get' } })

Reviewer: I feel `httpMethodUsed` is unnecessarily verbose. Better IMO would be just `method` or `httpMethod`.

###### Initial Metric List

> The cache can be initialized with a list of metrics name. It is useful when you already have the list of the metrics

Reviewer: ```suggestion
The cache can be initialized with a list of metric names. It is useful when you already have the list of the metrics
```

somewhere else in your application, and you would like to share this list with the embedded Prometheus client
of `codemirror-promql`.

Note: keep in mind that this list will be kept into the embedded Prometheus client until the time defined by `maxAge`.

```typescript
const promQL = new PromQLExtension().setComplete({
    remote: {
        cache: {

>             initialMetricList: ['ALERTS',

Reviewer: Wouldn't it be nicer to have `'ALERTS',` on its own line as well?

###### Initial Metric List

> The cache can be initialized with a list of metrics names. It is useful when you already have the list of the metrics

Reviewer: ```suggestion
The cache can be initialized with a list of metric names. It is useful when you already have the list of the metrics
```
